Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -238,3 +238,34 @@ parameters:
 ```
 
 * PVCs created using the new storageclass will be encrypted.
+
+## Enable Read affinity for RBD volumes
+
+Ceph CSI supports mapping RBD volumes with krbd options to allow
+serving reads from an OSD in proximity to the client, according to
+OSD locations defined in the CRUSH map. The format of the krbd option
```

Comment:
Can we not tell them the krbd format? It's an implementation detail, they can find it in the link to the krbd doc for more details.
```suggestion
OSD locations defined in the CRUSH map and topology labels on nodes.
```